Sprint_Tests/sofia_zahoruiko.md

```diff
@@ -0,0 +1,18 @@
+1. He had a deep understanding of the layout and working with advanced technologies at the frontend.
+After a short break of several years and a deep depression, plus listening to heavy music, he has poorly understanding of the modern trends and drinks a lot.
+If he could handle depression after his wife and kid death, he wouldn't have to find a job with a relevant salary now.
+
+2. If he had a goal and adequate salary expectations, he would have already working on the remote in the USA or in Bali with the prospect of moving to the Canary Islands and getting Spanish citizenship. 
+His dream from childhood is marrying Spanish and retiring early and growing grapes on the Balearic coast. 
+
+3. If you don't know how to work piece of code written by your colleague, you would have to write console.log everywhere and check the logical sequence.
+This is a labor-intensive task. If we had a documentation for this feature or a ticket for the implementation this feature with a description of the business logic, we would have to commit fix within a couple of hours.
```

Comment:
`If you don't know how the piece of code, written by your colleague, works, you will have to` - first conditional but zero conditional even better.
`cover with console logs`
`logical chain`
`time consuming task`
`documentation` is na uncountable word
`implementation of this feature` - missed the preposition
`we could commit the fix` -